lookout/style/format/postprocess.py

```diff
@@ -61,9 +65,39 @@ def filter_uast_breaking_preds(
     :return: List of predictions indices that are considered valid i.e. that are not breaking
              the UAST.
     """
+
+    def parse_partial_code(parent: bblfsh.Node, content: str
+                           ) -> Optional[Tuple[bblfsh.Node, int, int]]:
+        descendants = []
```

Comment:
Drive-by: There's quite a bit going on in this function and the nesting makes it difficult to understand the flow. Would you be willing to consider hoisting this to the top level and equipping it with a docstring?

Reply:
:+1: